src/common/options/rgw.yaml.in

```diff
@@ -3240,6 +3240,20 @@ options:
   - rados
   - dbstore
   - motr
+- name: dbstore_db_path
+  type: str
+  level: advanced
+  desc: experimental Option to set db backend store file path
```

Comment:
why experimental?  this seems like a fairly plausible option I'd look for

Reply:
yes..overlooked that part in copy/paste. will fix it.